### ISSUE TITLE
Extension fails to start due to CSP error

### ISSUE THREAD

**Issue by Spurlos** [Haaxor1689/humble-steam-sync#22]
Using Chrome Beta 130 and I have started getting the following error while the extension tires to start:

```
Refused to load the script 'chrome-extension://6f71604d-c584-491e-9c77-58aca79e2ce1/assets/main.tsx-f6dd0cda.js' because it violates the following Content Security Policy directive: "script-src 'self' 'wasm-unsafe-eval' 'inline-speculation-rules'". Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback.
main.tsx-loader-971cff2c.js:8
```

It seems a misconfiguration of Manifest V3 for the CSP policies.
![image](https://github.com/user-attachments/assets/335a49d4-2bab-4766-bf21-0010cdd4f5d1)

Unfortunately i can't go back to Chrome stable, but I believe it has\will get the same behaviour when v130 lands in stable.

**Comment by Spurlos** [comment]
Core issue documented at https://github.com/crxjs/chrome-extension-tools/issues/918

Example fix during build process from a similar extension: https://github.com/Zortrox/steamchecker/commit/f06f4b2475240da29f2c7ae0153b7808f0d2b402#diff-2f655a9559f7bbabd6e1f6035dc69cf3e3078491d348ecc40aefd178be735506R2

**Comment by richard-jr-lewis** [comment]
I can confirm the above issue. I am not on a Chrome Beta. This is happening on version 130.0.6723.70.

**Comment by Haaxor1689** [comment]
thank you for reporting, I'll hopefully be able to get to it in coming days, was super busy past few months

**Comment by richard-jr-lewis** [comment]
No worries, I appreciate the hard work you've put into this.

**Comment by Spurlos** [comment]
> thank you for reporting, I'll hopefully be able to get to it in coming days, was super busy past few months

Busy dev here also. I feel you! Keep up the good work!

**Comment by Spurlos** [comment]
The core issue has been fixed in the https://github.com/crxjs/chrome-extension-tools/issues/918

**Comment by richard-jr-lewis** [comment]
I can see it's been fixed in 3.1. This issue can be closed. 🎉

**Comment by Haaxor1689** [comment]
thank you for verifying :)
